1.7/administration/upgrading.md

```diff
@@ -0,0 +1,157 @@
+---
+post_title: Upgrading
```

Comment:
@kaysoky those fields are optional. the default title of this topic is already brief so i think nav_title is unnecessary.  however, menu_order is important here - i have added.